test/vstest.console.UnitTests/TestPlatformHelpers/TestRequestManagerTests.cs

```diff
@@ -6,106 +6,106 @@
 
 namespace vstest.console.UnitTests.TestPlatformHelpers
 {
-    using System;
```

Comment:
what is going on in this file? was it using Tabs as spaces?

Reply:
hard to see from this view but the file has few thousand changes.

Reply:
Yes so i used the [FixMixedTabs](https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.FixMixedTabs) plugin to remove spaces with tabs, so looks like the plugin rewrites the whole cs file.